vcr/holder/openid.go

```diff
@@ -200,7 +200,7 @@ func (h *openidHandler) retrieveCredential(ctx context.Context, issuerClient ope
 	claims := map[string]interface{}{
 		"aud":   issuerClient.Metadata().CredentialIssuer,
 		"iat":   nowFunc().Unix(),
-		"nonce": tokenResponse.CNonce,
+		"nonce": *tokenResponse.GetString("c_nonce"),
```

Comment:
panic

Reply:
checked in the calling func. But i'll change it to an empty string